### PR TITLE
Implement get_composite_mode on DBus-Api

### PR DESCRIPTION
**Pull request by MaZderMind**
Remote-Applications such as the Python-UI can retrieve the current Composition-Mode this way

### CODE REVIEW

File: python-api/gstswitch/connection.py [hunk 1]
```diff
@@ -282,6 +282,29 @@ def set_composite_mode(self, mode):
...
+            result = connection.call_sync(
+                self.bus_name,
+                self.object_path,
+                self.default_interface,
+                'get_composite_mode',
+                None,
+                GLib.VariantType.new("(i)"),
+                Gio.DBusCallFlags.NONE, -1,
+                None)
+            return result
```
Comment (mithro) [inline python-api/gstswitch/connection.py]: Looks like you're working from an older version of this file. These should probably be mapped to the names at https://github.com/timvideos/gst-switch/blob/master/python-api/gstswitch/controller.py#L21 in some way?

**Comment by mithro** [comment]
Tests?

**Comment by MaZderMind** [comment]
I'm unsure where the names you mentioned (I got the mail but can't find the message on github) should come into play. I copied most of the code from set_composite_mode and found the names in master not being used there either.

**Comment by mithro** [comment]
Sorry that was my fault - shouldn't do code reviews when it's 2am :)

Those names are used in the controller.py not the connection.py file.

**Pull request merged**